lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_pipeline_upload_action.rb

```diff
@@ -1,17 +1,20 @@
 module Fastlane
   module Actions
     class BuildkitePipelineUploadAction < Action
-      DEFAULT_ENV_FILE = File.join('.buildkite', 'shared-pipeline-vars').freeze
+      DEFAULT_BUILDKITE_PIPELINE_FOLDER = '.buildkite'.freeze
+      DEFAULT_ENV_FILE = File.join(DEFAULT_BUILDKITE_PIPELINE_FOLDER, 'shared-pipeline-vars').freeze
 
       def self.run(params)
         pipeline_file = params[:pipeline_file]
+        pipeline_file = File.join(DEFAULT_BUILDKITE_PIPELINE_FOLDER, pipeline_file) unless File.absolute_path?(pipeline_file)
         env_file = params[:env_file]
-        environment = params[:environment]
+        # Both keys and values need to be passed as strings
+        environment = params[:environment].to_h { |k, v| [k.to_s, v.to_s] }
```

Comment:
```suggestion
        # Both keys and values need to be passed as strings otherwise Fastlane.sh will fail to parse the command.
        environment = params[:environment].to_h { |k, v| [k.to_s, v.to_s] }
```

Just a nitpick to leave a note regarding the reason why these need to be strings. I'd like to keep pushing the fix so it eventually goes to Fastlane itself but... _priorities_.